lms/djangoapps/teams/tests/test_views.py

```diff
@@ -129,7 +129,7 @@ def test_query_counts(self):
         team.add_user(self.user)
 
         # Check the query count on the dashboard again
-        with self.assertNumQueries(19):
+        with self.assertNumQueries(20):
```

Comment:
Hmm.. thanks for researching that @wajeeha-khalid.  FYI @cahrens, T&L may want to consider reducing the number of SQL calls for the team dashboard by optimizing the number of SQL queries necessary for rendering a single dashboard.  I wonder if it grows linearly with the number of topics, which it shouldn't need to.